### PR TITLE
home about us section and branding section completed

### PR DESCRIPTION
**Pull request by hamzaussaid9**

### CODE REVIEW

File: client/src/controls/ThreeDotButton.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ThreeDotButton = () => {
```
Comment (safiullah7) [inline client/src/controls/ThreeDotButton.tsx:3]: please remove this component

File: client/src/features/FAQs/FAQs.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+const FAQs = () => {
+    return (
+        <>
+            <FAQsContainer>
+                <Container maxWidth="xl" className="container" >
+                    <FAQsContent>
+                        <h2>Frequently Asked Questions</h2>
+                    </FAQsContent>
+                    <FAQsQAsList />
```
Comment (safiullah7) [inline client/src/features/FAQs/FAQs.tsx:14]: get data in this component and pass into the child component

File: client/src/features/FAQs/FAQsQAsItem.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                    </IconButton></h3>
+                <div className={showAnswer ? 'show-answer' : 'hide-answer'}>
+                    <p ref={refShow}>{answer}</p>
+                </div>
+            </FAQsQAsListItem>
+        </>
+    )
+}
+
+interface IPropsSTyled {
```
Comment (safiullah7) [inline client/src/features/FAQs/FAQsQAsItem.tsx:42]: correct spellings please

File: client/src/features/FAQs/FAQsQAsList.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components';
+import { faqsQAs } from '../../models/faqs';
+import FAQsQAsItem from './FAQsQAsItem';
```
Comment (safiullah7) [inline client/src/features/FAQs/FAQsQAsList.tsx:4]: remove data-getting from here and get data from props

File: client/src/features/home/BriefAboutUsSection.tsx [hunk 5]
```diff
@@ -1,7 +1,157 @@
...
+              <Grid item md={4} sm={6} xs={12}>
+                <GridDrafting>
+                  <img src="about-draft.png" alt="about-draft" />
+                  <h3>Document Drafting</h3>
+                  <p>
+                    We draft/review any tech and non-tech contracts including NDAs, SLAs, IP Lisening/Assignment, EULAs, etc.
+                  </p>
+                </GridDrafting>
+              </Grid>
+              <Grid item md={4} sm={6} xs={12}>
```
Comment (safiullah7) [inline client/src/features/home/BriefAboutUsSection.tsx:43]: update styled component names

File: client/src/controls/ThreeDotButton.tsx (hunk 1 above)
Comment (hamzaussaid9) [reply client/src/controls/ThreeDotButton.tsx]: has been removed

File: client/src/features/FAQs/FAQs.tsx (hunk 2 above)
Comment (hamzaussaid9) [reply client/src/features/FAQs/FAQs.tsx:14]: names has been changed

File: client/src/features/FAQs/FAQsQAsItem.tsx (hunk 3 above)
Comment (hamzaussaid9) [reply client/src/features/FAQs/FAQsQAsItem.tsx]: spelling has been corrected

File: client/src/features/FAQs/FAQsQAsList.tsx (hunk 4 above)
Comment (hamzaussaid9) [reply client/src/features/FAQs/FAQsQAsList.tsx]: Updated

**Review by safiullah7 — APPROVED** [review]

**Pull request merged**